src/block.ts

```diff
@@ -303,13 +373,34 @@ export class TimelineBlockProcessor {
         }
 
         createInternalLinkOnNoteCard( eventAtDate, noteCard )
-        noteCard.createEl( 'p', { text: eventAtDate.body.trim() })
+        eventAtDate.body && noteCard.createEl( 'p', { text: eventAtDate.body.trim() })
+      }
+
+      /* If the previous note is of class 'timeline-tail' then there's a chance that the current node belongs nested within
+         the previous one. Here we iterate backwards for as long as elements of the class 'timeline-tail' are encountered.
+
+         Then we sort an array containing the start and ending date of the current event along with the ending date of the
+         previous event. If the index of the ending date of the previous event in the new array is greater than 0 then the
+         note(s) preceding it (either the note belonging to the start date or the notes belonging to both the start and end
+         dates of the current event) are placed before their predecessor. */
+      for( let s = [...timeline.children],i = s.indexOf( noteDiv[0] ); s[i-1]?.classList.contains( 'timeline-tail' ); i-- ) {
+        const t = s[i-1].getAttribute( 'timeline-date' )
+        for( let j = [start, end, t].sort().lastIndexOf( t ); j > 0; s[i-1].before( ...noteDiv.slice( 0, j )), j = 0 ) {
```

Comment:
`...noteDiv.slice( 0, j )` ensures that the start note is prepended before the previous end note AS WELL AS the end note iv this event has one and it is appropriate to do so. So the code allows for one spanning event to span outside of another which it begins within, but graphically I don't know how to handle that